### ISSUE TITLE
unknown document: plot_seed_to_voxel_correlation

### ISSUE THREAD

**Issue by chrisfilo** [nilearn/nilearn#1388]
```
/home/ubuntu/nilearn/nilearn/image/resampling.py:472: UserWarning: Casting data from int64 to float64
  warnings.warn("Casting data from %s to %s" % (data.dtype.name, aux))
/home/ubuntu/miniconda2/envs/testenv/lib/python3.5/site-packages/scipy/ndimage/interpolation.py:430: UserWarning: The behaviour of affine_transform with a one-dimensional array supplied for the matrix parameter has changed in scipy 0.18.0.
  "The behaviour of affine_transform with a one-dimensional "
/home/ubuntu/miniconda2/envs/testenv/lib/python3.5/site-packages/sklearn/utils/validation.py:526: DataConversionWarning: A column-vector y was passed when a 1d array was expected. Please change the shape of y to (n_samples, ), for example using ravel().
  y = column_or_1d(y, warn=True)
/home/ubuntu/miniconda2/envs/testenv/lib/python3.5/site-packages/numpy/lib/function_base.py:3858: RuntimeWarning: Invalid value encountered in median
  r = func(a, **kwargs)

Traceback (most recent call last):
  File "/home/ubuntu/miniconda2/envs/testenv/lib/python3.5/site-packages/Sphinx-1.5.1-py3.5.egg/sphinx/cmdline.py", line 296, in main
    app.build(opts.force_all, filenames)
  File "/home/ubuntu/miniconda2/envs/testenv/lib/python3.5/site-packages/Sphinx-1.5.1-py3.5.egg/sphinx/application.py", line 333, in build
    self.builder.build_update()
  File "/home/ubuntu/miniconda2/envs/testenv/lib/python3.5/site-packages/Sphinx-1.5.1-py3.5.egg/sphinx/builders/__init__.py", line 251, in build_update
    'out of date' % len(to_build))
  File "/home/ubuntu/miniconda2/envs/testenv/lib/python3.5/site-packages/Sphinx-1.5.1-py3.5.egg/sphinx/builders/__init__.py", line 322, in build
    self.write(docnames, list(updated_docnames), method)
  File "/home/ubuntu/miniconda2/envs/testenv/lib/python3.5/site-packages/Sphinx-1.5.1-py3.5.egg/sphinx/builders/__init__.py", line 360, in write
    self._write_serial(sorted(docnames), warnings)
  File "/home/ubuntu/miniconda2/envs/testenv/lib/python3.5/site-packages/Sphinx-1.5.1-py3.5.egg/sphinx/builders/__init__.py", line 370, in _write_serial
    self.warn(*warning, **kwargs)
  File "/home/ubuntu/miniconda2/envs/testenv/lib/python3.5/site-packages/Sphinx-1.5.1-py3.5.egg/sphinx/application.py", line 399, in warn
    raise SphinxWarning(warntext)
sphinx.errors.SphinxWarning: /home/ubuntu/nilearn/doc/auto_examples/01_plotting/plot_surf_stat_map.rst:28: WARNING: unknown document: plot_seed_to_voxel_correlation


Warning, treated as error:
/home/ubuntu/nilearn/doc/auto_examples/01_plotting/plot_surf_stat_map.rst:28: WARNING: unknown document: plot_seed_to_voxel_correlation
```

More details: https://circleci.com/gh/nilearn/nilearn/2276 I believe this was introduced in #1387.

**Comment by chrisfilo** [comment]
Still failing https://circleci.com/gh/nilearn/nilearn/2278

**Comment by chrisfilo** [comment]
Should I turn on CircleCI integration for forks and PRs so such things would be caught during PR review?

**Comment by GaelVaroquaux** [comment]
Yes, it would be nice. It vanished a little while ago 

⁣Sent from my phone. Please forgive brevity and mis spelling​

**Comment by chrisfilo** [comment]
Done - let me know if you don't see it for newly opened PRs.

**Comment by KamalakerDadi** [comment]
Thanks @chrisfilo . CircleCI is now ON. This issue will be fixed with this PR #1392

**Comment by GaelVaroquaux** [comment]
That's fixed, isn't it?

**Comment by KamalakerDadi** [comment]
Yes fixed in master as CircleCI is green.

**Comment by GaelVaroquaux** [comment]
Great. Can you close it, please.

**Comment by KamalakerDadi** [comment]
Thanks for reporting @chrisfilo . Closing as it is fixed in master with this PR #1392 . Let me know if you still see errors.
